standard/info/wikis/formula1/info.lua

```diff
@@ -0,0 +1,31 @@
+---
+-- @Liquipedia
+-- wiki=formula1
+-- page=Module:Info
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	startYear = 1950,
+	wikiName = 'formula1',
+	name = 'Formula 1',
+	games = {
+		formula1 = {
+			abbreviation = 'F1',
+			name = 'Formula 1',
+			link = 'Formula 1',
+			logo = {
+				darkMode = 'F1 2018 allmode.png',
+				lightMode = 'F1 2018 allmode.png',
+			},
+			defaultTeamLogo = {
+				darkMode = 'Sim Racing default darkmode.png',
```

Comment:
really?

Reply:
its a standard racing wheel icon, we could get a custom one made or find an existing?